src/index.js

```diff
@@ -6,17 +6,26 @@ client.collectDefaultMetrics();
 
 module.exports = {
     client,
-    middleware: async (ctx, next) => {
-        ctx.state.prometheus = client;
-        if (ctx.path === '/metrics') {
-            debug('GET /metrics');
-            if (!ctx.headers["x-forwarded-for"]) {
-                ctx.set('Content-Type', client.register.contentType);
-                ctx.body = client.register.metrics();
-            } else {
-                ctx.throw(403, "Forbidden");
+    middleware: (options = {}) => {
+        const path = options.path || '/metrics';
+        const [headerBlacklist] = [options.headerBlacklist];
```

Comment:
This line is a bit strange, feels like it should be `const { headerBlacklist } = options;`